drivers/sdhc/imx_usdhc.c

```diff
@@ -376,11 +376,20 @@ static int imx_usdhc_set_io(const struct device *dev, struct sdhc_io *ios)
 			imxrt_usdhc_pinmux(cfg->nusdhc, false, 2, 7);
 #endif
 			break;
+		case SDHC_TIMING_HS400:
+#if FSL_FEATURE_USDHC_HAS_HS400_MODE
```

Comment:
This commit should be moved to before the HS400 support in the eMMC protocol imo, since it is required for HS400 support to work on the integration platform being used to test it.